src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/localize/templatestrings.cs.json

```diff
@@ -5,10 +5,8 @@
   "symbols/Framework/description": "The target framework for the project.",
   "symbols/Framework/choices/net7.0/description": "Target net7.0",
   "symbols/skipRestore/description": "If specified, skips the automatic restore of the project on create.",
-  "symbols/Hosted/displayName": "ASP.NET Core _Hosted",
+  "symbols/Hosted/displayName": "ASP.NET Core Hosted",
```

Comment:
We'll actually want to keep this as-is. `_` is a hot key (?) (I forget the official term), but it allows users to press a key in VS to quickly navigate to the option without using their mouse.